src/LayerPlan.cpp

```diff
@@ -1064,7 +1064,17 @@ unsigned LayerPlan::locateFirstSupportedVertex(ConstPolygonRef wall, const unsig
     }
 }
 
-void LayerPlan::addLinesByOptimizer(const Polygons& polygons, const GCodePathConfig& config, SpaceFillType space_fill_type, bool enable_travel_optimization, int wipe_dist, float flow_ratio, std::optional<Point> near_start_location, double fan_speed)
+void LayerPlan::addLinesByOptimizer
+(
+    const Polygons& polygons,
+    const GCodePathConfig& config,
+    SpaceFillType space_fill_type,
+    bool enable_travel_optimization,
+    int wipe_dist,
```

Comment:
Since `wipe_dist` is a distance, it should be a `coord_t`.

Reply:
I only changed the formatting here, but you're right!